### PR TITLE
UPTM-13 - Dashboard Display on vertical 4K screen

### PR DESCRIPTION
**Pull request by gitstart**
Here is the GitStart Ticket for this pull request: [UPTM-13](https://developers.gitstart.com/client/Uptime-Kuma/ticket/UPTM-13)

- Please mark your PR as "ready for review" when you have completed all requirements.
- Please ensure that your PR is reviewed by at least 2 people after you have completed it.
- Comments for review or requested changes from the upstream client will be notified to you by email and will also show on the [ticket page](https://developers.gitstart.com/client/Uptime-Kuma/ticket/UPTM-13).
- After the PR is merged on the upstream, please merge this PR.
- If there exist merge conflicts on your current PR and you have to close the PR instead of merging it, please go to the [ticket page](https://developers.gitstart.com/client/Uptime-Kuma/ticket/UPTM-13) to manually finish the task that is related to this PR.

### Loom video
https://www.loom.com/share/6fd4a2d46b18430c832ea7b67a6070f5
Happy hacking! 🎉

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.

### CODE REVIEW

File: src/pages/DashboardHome.vue [hunk 1]
```diff
@@ -55,15 +55,19 @@
                         </tr>
                     </tbody>
                 </table>
-
-                <div class="d-flex justify-content-center kuma_pagination">
+                <div v-if="!showAll" class="d-flex justify-content-center kuma_pagination">
```
Comment (CommanderStorm) [inline src/pages/DashboardHome.vue:58]:
```suggestion

                <div v-if="!showAll" class="d-flex justify-content-center kuma_pagination">
```

File: src/pages/DashboardHome.vue [hunk 2]
```diff
@@ -55,15 +55,19 @@
...
                     <pagination
                         v-model="page"
                         :records="importantHeartBeatList.length"
                         :per-page="perPage"
                         :options="paginationConfig"
                     />
                 </div>
+                <div v-if="importantHeartBeatList.length > perPage" class="d-flex justify-content-center my-1">
+                    <button class="btn btn-link" @click="toggleShowAllRecords">
+                        {{ showAll ? $t("Show Less") : $t("Show All") }}
```
Comment (CommanderStorm) [inline src/pages/DashboardHome.vue:68]: In general:
please add additional translation keys to the `en.json` (and any other language for which you are a native speaker) file.
If they are not available in this file, they won't be [available for translation on weblade](https://github.com/louislam/uptime-kuma/blob/master/src/lang/README.md).

**Review by CommanderStorm — CHANGES_REQUESTED** [review]
I have two requests:
- your changes seem to include other pull requests.
  Could you rebase your work onto master?
  This would make code reviews a lot easier.
- While I accept that on large devices our UI looks quite empty, I don't fully agree that the proposed solution is the best solution.
  - The reason why I think this is not the best solution is, because your button placement seems very out of place. (If I were a user, I would not expect there to be a `show all` button in the pagination selector)
  - I think that the `perPage` variable being calculated based on available vertical space would be a better solution.

**Comment by louislam** [comment]
Rebase and force push as it contained some commits which were not belong to this pr.

**Comment by louislam** [comment]
I am appreciated that the @gitstart team have made several pull requests for Uptime Kuma. Unfortunately, I agree with @CommanderStorm. It is not something I want to add.

Since it didn't follow the pr template, I hope your team have read our rules. I really don't want to waste anyone's time, so better ask me first next time.
```
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]:

I have read and understand the pull request rules.
```

**Pull request closed without merging**